packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -4175,7 +4175,7 @@ export default class Meeting extends StatelessWebexPlugin {
       return Promise.reject(new UserNotJoinedError());
     }
     // If the user is unjoined or guest waiting in lobby dont allow the user to addMedia
-    if (this.guest && MeetingUtil.isUserInIdleState(this.locusInfo) && !this.wirelessShare) {
+    if (MeetingUtil.isUserInIdleState(this.locusInfo) && !this.wirelessShare) {
```

Comment:
this change implies that locus self.state being IDLE means that user is in a lobby. Has this been confirmed with Locus team or maybe there are some Locus docs that confirm this?

Reply:
@arun3528 Could you please confirm this?

Reply:
Added check for self.state being IDLE and intent type WAIT as per the confirmation from locus team.